### DISCUSSION TITLE
How can i make a PR to github (the web page)

### DISCUSSION THREAD

**Discussion by amaimus** [community/community#69682] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

I want to know how I can fork the GitHub web page repository to create a pull request (PR). There is a change I want to propose, but I can't even find the repository.

The change I want to propose is specifically related to this view:
(Im using one of my repositories just as an example)
https://github.com/amaimus/react-creating-router-package

So, I need to know if the repository is open source (I assume it is) and which one it is.

**Comment by santoshvandari** [comment]
To create a pull request (PR) on the GitHub web page for a repository, you'll need to follow these general steps:

1. Fork the Repository:

   - Visit the repository's page on GitHub. In your example, it's: https://github.com/amaimus/react-creating-router-package.
   - In the upper-right corner of the repository's page, click the "Fork" button. This will create a copy (fork) of the repository under your GitHub account.
2. Clone Your Fork:

   - Once you have forked the repository, navigate to your own GitHub account's repositories. You should see a copy of the repository you just forked there.
   - Click on your forked repository to open it.
   - Click the "Code" button, and then copy the repository's URL (it will be in the format https://github.com/your-username/repository-name.git).
3. Clone the Repository Locally:

    - Open a terminal or command prompt on your local machine.
    - Use the git clone command to clone your forked repository to your local machine. Replace repository-url with the URL you copied in the previous step.
      ```bash
      git clone repository-url
      ```
4. Make Changes:

    - Make the desired changes to the code or files in your local clone of the repository.
5. Commit Your Changes:

    - Use the git add and git commit commands to stage and commit your changes to your local repository.
6. Push Changes to Your Fork:

     - Use the git push command to push your changes to your forked repository on GitHub.
7. Create a Pull Request:

   -  Visit your forked repository on GitHub.
   -  Click the "Pull Requests" tab.
   -  Click the "New Pull Request" button.
    - GitHub will compare the changes between your forked repository and the original repository.
    - Add a title and description for your pull request, and then click the "Create Pull Request" button.
8. Review and Submit:
   -  Review your pull request and ensure that the changes look correct.
   - If everything is in order, click the "Create Pull Request" button again to submit your PR.

Your pull request will now be submitted to the original repository for review. The maintainers of the repository will have the opportunity to review your changes and merge them if they find them suitable.

Please note that not all repositories are open source, so ensure that the repository you are forking and making changes to allows for contributions from external contributors.

**Reply by amaimus** [reply]
###  Hi!
Thanks for your reply @santoshvandari !

I expressed myself poorly when presenting my question, I apologize.

The process you mentioned is, correctly, the way to make a PR to a github repository

What I would really like to know is how to make a PR to the github repository (github.com)

For example, imagine that I wanted to change the order of the tabs in the repository view
![image](https://github.com/community/community/assets/35699916/22cb10c0-42e9-4b44-8314-b6f5e026da59)

To do this, I would have to clone the github repository, make the changes, and create a PR

### I would like to know:

What is the repository and if it is open source

**Reply by santoshvandari** [reply, marked as answer]
I apologize for not understanding your question well. 
GitHub offers a platform for hosting open-source projects, but the GitHub platform itself, including its core infrastructure and user interface, is not open-source. GitHub provides the tools and services to host and collaborate on open-source projects, but the underlying code that powers the GitHub platform is proprietary and managed by GitHub, Inc.

Users and developers can contribute to open-source projects hosted on GitHub and provide feedback and suggestions for improving the GitHub platform through the built-in feedback mechanisms. However, making direct code contributions to the GitHub platform itself is not an option for external contributors because the platform's codebase is not open-source.

In summary, while you can contribute to open-source projects on GitHub, the GitHub platform itself is not open-source, and contributions to its core codebase are not possible for external contributors.

**Reply by amaimus** [reply]
Thank you so much! That was exactly the doubt I had, I've been wondering for several days.
Thank you for your diligent response 😊
